app/models/manageiq/providers/container_manager/orchestration_stack.rb

```diff
@@ -0,0 +1,58 @@
+class ManageIQ::Providers::ContainerManager::OrchestrationStack < ::OrchestrationStack
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::ContainerManager"
+  belongs_to :container_template, :foreign_key => :orchestration_template_id, :class_name => "ContainerTemplate"
+
+  def self.create_stack(container_template, params, project_name)
+    new(:name                  => container_template.name,
```

Comment:
I saw the other PR you created `OrchestrationStack` for OpenShift and its `Status` class. It then makes sense to revert your earlier code to use the `Status` subclass, but here you need to create the stack instance with the right class.